test-harness/index.ts

```diff
@@ -44,10 +44,30 @@ describe('harness', () => {
       const [command, ...args] = parsed.command.split(' ').map(t => t.trim());
       const serverArgs = [...parsed.server.split(' ').map(t => t.trim()), tmpFileHandle.name];
 
-      prismMockProcessHandle = spawn(path.join(__dirname, '../cli-binaries/prism-cli'), serverArgs);
+      const spawnCommand = process.env.FROM_SOURCES
+        ? 'yarn'
+        : path.join(__dirname, '../cli-binaries/prism-cli');
+
+      prismMockProcessHandle = spawn(
+        spawnCommand,
+        process.env.FROM_SOURCES
+          ? ['workspace', '@stoplight/prism-cli', 'run', 'cli'].concat(serverArgs)
+          : serverArgs,
+        {
+          env: process.env.FROM_SOURCES
+            ? {
+                ...process.env,
+                NODE_OPTIONS: '--inspect',
+              }
+            : undefined,
+          stdio: 'pipe',
+        }
+      );
 
       prismMockProcessHandle.stdout.pipe(split2()).on('data', (line: string) => {
         if (line.includes('Prism is listening')) {
+          console.log(parsed.test);
```

Comment:
Leftover?

Reply:
This is kind of on purpose. Unfortunately the way the harness is run is "all or nothing" (there are technical reasons for this) and so jest can't really display what particular test is running right now.

With this hack, you can see the executed test.